views/layouts/main.handlebars

```diff
@@ -51,8 +51,8 @@
         <div id="navbarBasicExample" class="navbar-menu">
             <div class="navbar-end">
                 <div class="navbar-item has-dropdown is-hoverable">
-                    <a class="navbar-link nohoverstyle" href="/active-clubs">
-                        Book Clubs
```

Comment:
keep the one with 'nohoverstyle' and 'active-clubs' please

Reply:
added back in!